docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+= Configuring KubeArchive
+
+This document explains how to configure KubeArchive so you can archive 
+resources and query them.
+
+== Prerequisites
+* KubeArchive installed and running in a cluster (xref:getting-started/installation.adoc[])
+* Know how to query the KubeArchive API (xref:getting-started/kubearchive-api.adoc[])
+
+== The KubeArchiveConfig resource
+
+To configure KubeArchive to archive or delete resources create a
+KubeArchiveConfig custom resource. KubeArchiveConfigs are limited to 1 per namespace
+and should be named `kubearchive`. KubeArchiveConfigs have this general form:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources: [...]
+----
+
+`spec.resources` is a list of elements, each defining rules for a specific kind, so KubeArchive knows
+when to archive or delete them.
+
+==  `selector`: Selecting Resources
```

Comment:
I don't think we should document this at all, but rather point to the `ApiServerSource` documentation for this.

Otherwise we are duplicating things and have to keep them in sync.

Reply:
I think pointing them to Knative's documentation will be more confusing as they will assume we support Knative's ApiServerSource `selector`, which allows for `matchLabels`, `matchExpression`... However we don't use that when we process the KubeArchiveConfig and create an ApiServerSource. We are only using the `kind` and `apiVersion` (func `parseConfigMap` in the operator code).